lib/v2/components/snack_bar_info.dart

```diff
@@ -16,4 +16,8 @@ class SnackBarInfo extends SnackBar {
             ],
           ),
         );
+
+  void show(BuildContext context) {
```

Comment:
little helper i added to snackbarinfo

Reply:
Great!

This is what the experience brings, I had not found how to save that boilerplate.

I have to test it if it does not have drawbacks, everything is fine :D